stripe-core/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+    <!--
+    See https://developer.android.com/training/package-visibility/declaring for more details.
+    -->
+    <queries>
+        <!--
+        Added to check the browsers support Custom Tabs Service.
+        -->
+        <intent>
```

Comment:
seems this adds a service to all SDKs, is it needed for compile? could we only declare it at sdk level?